### DISCUSSION TITLE
Welcome to GitHub Certifications Discussions!

### DISCUSSION THREAD

**Discussion by tomwechsler** [tomwechsler/GitHub_Certifications#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by celebimh** [comment]
Hello to everyone. I am Mehmet.  I want to improve myself in the field of knowledge in using git and github.  I get information from different sources and I do experiments in the lab to practice.
